### PR TITLE
docs: Swap order of HCL2/JSON example code tabs so that the HCL2 tab is the default

### PR DESCRIPTION
**Pull request by aidan-mundy**
Swaps order of HCL2/JSON example code tabs so that the HCL2 tab is the default

Ran `make generate` after committing changes, which did not result in a diff, so I believe this should be sufficient.

Note: The diff looks weird because git diff is getting confused by the similar blocks of code. For easier reading, I split the changes into two commits where the swaps aren't being confused by git diff.

### CODE REVIEW

File: docs/builders/docker.mdx [hunk 1]
```diff
@@ -609,6 +609,25 @@ Gallery](https://gallery.ecr.aws/). The post processors work as described above
...
+    post-processors {
+        post-processor "docker-tag" {
+            repository = "public.ecr.aws/YOUR REGISTRY ALIAS HERE/YOUR REGISTRY NAME HERE"
+            tags       = ["latest"]
+        }
+
+        post-processor "docker-push" {
+            "ecr_login": true,
+            "aws_access_key": "YOUR KEY HERE",
+            "aws_secret_key": "YOUR SECRET KEY HERE",
```
Comment (nywilken) [inline docs/builders/docker.mdx:624]: It looks like we might be mixing JSON with HCL here.

File: docs/builders/docker.mdx [hunk 2]
```diff
@@ -340,6 +340,20 @@ of post-processors that are treated as as single pipeline, see
 [Post-Processors](/packer/docs/templates/legacy_json_templates/post-processors) for more information):
 
 <Tabs>
+<Tab heading="HCL2">
+
+```hcl
+  post-processors {
+    post-processor "docker-import" {
+        repository =  "myrepo/myimage"
+        tag = "0.7"
```
Comment (nywilken) [inline docs/builders/docker.mdx:349]:
```suggestion
        tags = ["0.7"]
```

File: docs/post-processors/docker-import.mdx [hunk 3]
```diff
@@ -22,6 +22,27 @@ registry.
...
+}
+
+build {
+  sources = [
+    "source.docker.example"
+  ]
+
+  post-processor "docker-import" {
+    repository = "local/ubuntu"
+    tag = "latest"
```
Comment (nywilken) [inline docs/post-processors/docker-import.mdx:40]:
```suggestion
    tags = ["latest"]
```

**Review by nywilken — APPROVED** [review]
Thanks for making these fixes. It looks like we might have a mix of JSON and HCL code snippet and some inconsistent use of tags attributed. I would just spot check those and merge when ready.

File: docs/post-processors/docker-import.mdx [hunk 4]
```diff
@@ -95,6 +95,16 @@ is optional.
 An example is shown below, showing only the post-processor configuration:
 
 <Tabs>
+<Tab heading="HCL2">
+
+```hcl
+post-processor "docker-import" {
+  repository = "hashicorp/packer"
+  tag = "0.7"
```
[the remark this answers is not in the document]
Comment (aidan-mundy) [reply docs/post-processors/docker-import.mdx:103]: The `docker-import` post-processor only accepts a single tag, from the `tag` (`string`) field. The tag post-processor allows multiple with the `tags` (`list[string]`) field or the `tag` (`list[string]`) field (maintained for backward compatibility.

File: docs/post-processors/docker-import.mdx (hunk 3 above)
Comment (aidan-mundy) [reply docs/post-processors/docker-import.mdx:40]: The `docker-import` post-processor only accepts a single tag, from the `tag` (`string`) field. The tag post-processor allows multiple with the `tags` (`list[string]`) field or the `tag` (`list[string]`) field (maintained for backward compatibility.

File: docs/builders/docker.mdx (hunk 2 above)
Comment (aidan-mundy) [reply docs/builders/docker.mdx:349]: The `docker-import` post-processor only accepts a single tag, from the `tag` (`string`) field. The tag post-processor allows multiple with the `tags` (`list[string]`) field or the `tag` (`list[string]`) field (maintained for backward compatibility.

File: docs/post-processors/docker-import.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (nywilken) [reply docs/post-processors/docker-import.mdx:103]: Ah okay I confused the import with the tag. Mi culpa.

**Comment by aidan-mundy** [comment]
Just noticed after merging that this also closes #121

**Pull request merged**